scripts/flash/get-timestamp.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var encodeTag = require("../../src/format/tag/encode-tag");
+
+function getTime(augur, auth, callback) {
+  var timestamp = augur.api.Controller.getTimestamp();
```

Comment:
Needs callback for async.

Reply:
converted the sync to async calls to get Controller getTimestamp()